### PR TITLE
feat(query): Added Authorization Mode Node Not Set for Kubernetes

### PR DESCRIPTION
**Pull request by cxMiguelSilva**
**Proposed Changes**
-Added Authorization Mode Node Not Set for Kubernetes

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by joaorufi — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/authorization_mode_node_not_set/metadata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "4d7ee40f-fc5d-427d-8cac-dffbe22d42d1",
+  "queryName": "Authorization Mode Node Not Set",
+  "severity": "MEDIUM",
+  "category": "Insecure Configurations",
+  "descriptionText": "The authorization-mode flag should have 'Node' mode",
```
Comment (rafaela-soares) [inline assets/queries/k8s/authorization_mode_node_not_set/metadata.json:6]:
```suggestion
   "descriptionText": "When using kube-apiserver command, the 'authorization-mode' flag should have 'Node' mode",
```

File: assets/queries/k8s/authorization_mode_node_not_set/query.rego [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	args := container.args
+	startswith(args[a], flag)
+	values := split(args[a], "=")[1]
+	hasValue(values, value)
+}
+
+hasValue(values, value) {
+	splittedValues := split(values, ",")
+	splittedValues[_] == value
+}
```
[the remark this answers is not in the document]
Comment (rafaela-soares) [reply assets/queries/k8s/authorization_mode_node_not_set/query.rego:40]: If you add it, please update the query `kics/assets/queries/k8s/authorization_mode_set_to_always_allow`

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/authorization_mode_node_not_set/query.rego (hunk 2 above)
[the remark this answers is not in the document]
Comment (rafaela-soares) [reply assets/queries/k8s/authorization_mode_node_not_set/query.rego:40]: If you add it, please update` kics\assets\queries\k8s\authorization_mode_set_to_always_allow`

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Pull request merged**